src/stacks-hierarchy/Root_TabNavigatorStack/Root_TabNavigatorStack.tsx

```diff
@@ -63,6 +65,21 @@ export const Root_TabNavigatorStack = () => {
     nextOnboardingScreen?.name && goToScreen(false, nextOnboardingScreen);
   }, [nextOnboardingSection?.initialScreen, goToScreen, navigation]);
 
+  const getProfileNotification = (): ThemeIconProps['notification'] => {
+    if (customerNumber === undefined) {
```

Comment:
Could this just be `!customerNumber`, or is it a good reason for explicitly checking against `undefined`?